src/map/skill.cpp

```diff
@@ -9812,6 +9816,10 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 		}
 		break;
 
+	case NPC_HALLUCINATIONWALK:
+		clif_skill_nodamage(src,bl,skill_id,skill_lv,sc_start(src,bl,type,100,skill_lv,skill_get_time(skill_id,skill_lv)));
+		break;
+
```

Comment:
This can be moved to line 7100~ with the other basic skills that make a `clif_skill_nodamage`.